flutter/shell/platform/tizen/tizen_renderer_gl.cc

```diff
@@ -0,0 +1,104 @@
+// Copyright 2025 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "tizen_renderer_gl.h"
+#include "flutter/shell/platform/tizen/flutter_tizen_engine.h"
+#include "flutter/shell/platform/tizen/tizen_view.h"
+#include "flutter/shell/platform/tizen/tizen_view_base.h"
+#include "flutter/shell/platform/tizen/tizen_window.h"
+
+namespace flutter {
+
+TizenRendererGL::TizenRendererGL() {}
+
+FlutterTransformation TizenRendererGL::GetTransformation() const {
+  return flutter_transformation_;
+}
+
+FlutterRendererConfig TizenRendererGL::GetRendererConfig() {
+  FlutterRendererConfig config = {};
+  config.type = kOpenGL;
+  config.open_gl.struct_size = sizeof(config.open_gl);
+  config.open_gl.make_current = [](void* user_data) -> bool {
+    auto* engine = static_cast<FlutterTizenEngine*>(user_data);
+    if (!engine->view()) {
+      return false;
+    }
+    return dynamic_cast<TizenRendererGL*>(engine->renderer())->OnMakeCurrent();
+  };
+  config.open_gl.make_resource_current = [](void* user_data) -> bool {
+    auto* engine = static_cast<FlutterTizenEngine*>(user_data);
+    if (!engine->view()) {
+      return false;
+    }
+    return dynamic_cast<TizenRendererGL*>(engine->renderer())
+        ->OnMakeResourceCurrent();
+  };
+  config.open_gl.clear_current = [](void* user_data) -> bool {
+    auto* engine = static_cast<FlutterTizenEngine*>(user_data);
+    if (!engine->view()) {
+      return false;
+    }
+    return dynamic_cast<TizenRendererGL*>(engine->renderer())->OnClearCurrent();
+  };
+  config.open_gl.present = [](void* user_data) -> bool {
+    auto* engine = static_cast<FlutterTizenEngine*>(user_data);
+    if (!engine->view()) {
+      return false;
+    }
+    return dynamic_cast<TizenRendererGL*>(engine->renderer())->OnPresent();
+  };
+  config.open_gl.fbo_callback = [](void* user_data) -> uint32_t {
+    auto* engine = static_cast<FlutterTizenEngine*>(user_data);
+    if (!engine->view()) {
+      return false;
+    }
+    return dynamic_cast<TizenRendererGL*>(engine->renderer())->OnGetFBO();
+  };
+  config.open_gl.surface_transformation =
+      [](void* user_data) -> FlutterTransformation {
+    auto* engine = static_cast<FlutterTizenEngine*>(user_data);
+    if (!engine->view()) {
+      return FlutterTransformation();
+    }
+    return dynamic_cast<TizenRendererGL*>(engine->renderer())
+        ->GetTransformation();
```

Comment:
The existing `engine->view()->GetFlutterTransformation();` is used in FlutterTizenView::OnRotate. Renderer's GetTransformation() always returns only identity matrix. Is there a reason why the Transformation function was changed?

Reply:
Because Transformation is part of GL render callback, I think it's better move to gl_renderer. but I didn't notice that the Transformation was changed in flutter tizen view. I revert the code to flutter tizen view now.